src/layer/BYOCLayer.ts

```diff
@@ -220,6 +237,9 @@ export class BYOCLayer extends AbstractSentinelHubV3Layer {
   }
 
   protected getCatalogCollectionId(): string {
+    if (this.lowResolutionCollectionId !== undefined) {
```

Comment:
Not sure about this. I suppose it will work as expected as long as low res collection in derived from high res collection.